tools/addRoomsToDirectory.ts

```diff
@@ -89,20 +90,20 @@ const bridge = new Bridge({
     domain: "rubbish",
     homeserverUrl: true,
     registration: true,
-    roomStore: options.store,
 });
 
+const discordstore = new DiscordStore(config.database ? config.database.filename : "discord.db");
+
 async function run() {
     try {
-        await bridge.loadDatabases();
+        await discordstore.init();
```

Comment:
with #378 this should go into that discorbot.init

Reply:
Ditto!

Reply:
also this still